src/basic/process-util.c

```diff
@@ -922,6 +922,64 @@ int ioprio_parse_priority(const char *s, int *ret) {
         return 0;
 }
 
+/* The cached PID, possible values:
+ *
+ *     == 0          → cache not initialized yet
```

Comment:
I'd prefer #defines giving names to the 0 and -1 values, then there's less need for the comments in the switch below.